src/js/main.js

```diff
@@ -363,6 +363,29 @@ if (document.body.dataset.page === "integrations") {
   }
 }
 
+// Add underline class to menu links on Mobile
+
+function setActiveMenuItem() {
+  const currentUrl = window.location.href;
+  const menuItems = document.querySelectorAll("nav a");
+
+  menuItems.forEach((item) => {
+    const itemHref = item.getAttribute("href");
+    const itemUrl = new URL(itemHref, currentUrl);
+
+    const isIntegrationsPage = itemUrl.pathname === "/integrations";
+    const isInIntegrationsSection = currentUrl.includes("/integrations/");
+
+    const shouldBeUnderlined =
+      isIntegrationsPage &&
+      (isInIntegrationsSection || window.location.pathname === "/integrations");
```

Comment:
We can make it more general by checking if the prefix matches, something like this:

```suggestion
  const currentPath = window.location.pathname;
  const menuItems = document.querySelectorAll("nav a");

  menuItems.forEach((item) => {
    const itemPathname = item.pathname;
    const shouldBeUnderlined = currentPath.startsWith(itemPathname);
```

Reply:
@jonatanklosko, after applying this change is not working for me anymore. =/ Is there something else I should do?